modules/Virtualizer_module/src/VirtualizerModule.cpp

```diff
@@ -352,6 +412,9 @@ bool VirtualizerModule::close()
     m_encodersInterface = nullptr;
     m_controlModeInterface = nullptr;
 
+    m_tfDriver.close();
+    m_tfPublisher = nullptr;
```

Comment:
```suggestion
    m_tfPublisher = nullptr;
    m_tfDriver.close();
```

Reply:
I am not sure also if `m_useTf` is set to false, reaching this point while closing causes segfault or will close normally.

Reply:
The close is supposed to be safe (i.e. if nothing was opened, it should simply return). The order of the expressions does not really matter.